src/rgw/rgw_common.h

```diff
@@ -162,6 +166,12 @@ extern PerfCounters *perfcounter;
 extern int rgw_perf_start(CephContext *cct);
 extern void rgw_perf_stop(CephContext *cct);
 
+extern int percentile_perf_start(CephContext *cct);
+extern void percentile_perf_stop(CephContext *cct);
+
+extern centile::CentileCollection *get_lat_centile;
+extern centile::CentileCollection *put_lat_centile;
+
```

Comment:
While this is OK for the first cut, we will want latency counters for all the APIs (i.e. GET, PUT, DELETE, HEAD, LIST)